api/member.go

```diff
@@ -241,3 +241,27 @@ func (pma *ProjectMemberAPI) Delete() {
 		return
 	}
 }
+
+//sysadmin has all privileges to all projects
+func listRoles(userID int, projectID int64) ([]models.Role, error) {
+	roles := make([]models.Role, 1)
+	isSysAdmin, err := dao.IsAdminRole(userID)
+	if err != nil {
+		return roles, err
```

Comment:
@wy65701436 I think returning an empty array is more reasonable, which can avoid nil pointer exception if user use it directly.